### ISSUE TITLE
您好，当我尝试运行train.py时：

### ISSUE THREAD

**Issue by licongguan** [bubbliiiing/dcgan-keras#1]
InternalError (see above for traceback): Blas GEMV launch failed:  m=8192, n=64
	 [[node dense_2/MatMul (defined at /home/glc/anaconda3/envs/tf1.13/lib/python3.6/site-packages/keras/backend/tensorflow_backend.py:1075) ]]
	 [[node loss/mul (defined at /home/glc/anaconda3/envs/tf1.13/lib/python3.6/site-packages/keras/engine/training.py:835) ]]

**Comment by bubbliiiing** [comment]
环境配置问题吧
